### PR TITLE
Use ImdbPie for obtaining imdb info on load_imdb_info

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/tv.py [hunk 1]
```diff
@@ -1265,71 +1264,55 @@ def load_from_indexer(self, tvapi=None):
...
+                imdb_id = result['imdb_id']
+                title = result['title']
+                year = result['year']
+                logger.log(u'Results: {0} {1} {2}'.format( title, year, imdb_id))
+                if title == self.name and year == self.startyear:
+                    logger.log(u'{0} {1} matches IMDb year {2} and title {3}, using {4} as the imdb_id for the show.'.format(self.name, self.startyear, title, year, imdb_id))
+                    self.imdb_id = imdb_id
+                elif title == self.name and year != self.startyear and not self.imdb_id:
+                    logger.log(u'{0} matches IMDb title {1} , using {2} as the imdb_id for the show.'.format(self.name, title, imdb_id))
+                    self.imdb_id = imdb_id
```
Comment (medariox) [inline medusa/tv.py]: This won't always work. Take as example the search for "X Factor". It will return:
```
{u'year': u'2011', u'imdb_id': u'tt1582461', u'title': u'The X Factor'}
{u'year': u'2008', u'imdb_id': u'tt1188980', u'title': u'X Factor'}
{u'year': u'2010', u'imdb_id': u'tt1709082', u'title': u'X Factor'}
{u'year': u'2006', u'imdb_id': u'tt0892254', u'title': u'X Factor'}
{u'year': u'2008', u'imdb_id': u'tt1194223', u'title': u'X Factor'}
{u'year': u'2011', u'imdb_id': u'tt1774260', u'title': u'X Factor'}
```
I want the X Factor from year 2006, but the loop will keep changing it until it gets to X Factor from year 2011. Also the title comparison should be case-insensitive and it's probably a good idea to make it a method like it was before e.g. ```title_to_imdb(titles)```.

File: medusa/tv.py [hunk 2]
```diff
@@ -1265,71 +1264,55 @@ def load_from_indexer(self, tvapi=None):
...
         }
 
         self.externals['imdb_id'] = self.imdb_id
 
-        if imdb_obj.get('runtimes'):
-            self.imdb_info['runtimes'] = re.search(r'\d+', imdb_obj['runtimes'][0]).group(0)
-
-        # Get only the production country certificate if any
-        if imdb_obj.get('certificates') and imdb_obj.get('countries'):
-            for certificate in imdb_obj['certificates']:
```
Comment (medariox) [inline medusa/tv.py]: Why did you remove the certificates?

**Comment by p0psicles** [comment]
This finished?

**Comment by medariox** [comment]
This PR is open since 3 weeks. I won't rebase it again.

**Comment by fernandog** [comment]
i can rebase. no problem at all

**Comment by medariox** [comment]
So di it.

**Comment by fernandog** [comment]
5s rebase. got love pycharm

**Comment by fernandog** [comment]
so, is this finished as pops asked?

**Comment by fernandog** [comment]
@medariox if is working why don't you approve and merge as its super PR?!

**Comment by medariox** [comment]
Because I basically rewrote it completely and would like someone to review it first. Also already told on Slack it was concluded, obviously it's not anymore since the rebase.

**Comment by fernandog** [comment]
sorry. didn't understood. how come is not concluded anymore?  rebase is done

**Comment by medariox** [comment]
Did you expect the plots to work just like that? Magic involved?

**Comment by fernandog** [comment]
i didn't know imdbpie code. you could said "its not finished anymore because plot is different in imdbpie" - instead of "obviously it's not anymore since the rebase." - not obvious to me. sorry
